docs/source/release/changelog/591.feature.rst

```diff
@@ -0,0 +1,4 @@
+Added :func:`squidpy.tl.var_by_distance` to calculate distances to user-defined anchor points.
+Stores the resulting design matrix in :attr:`adata.obsm`.
+Added :func:`squidpy.pl.var_by_distance` to visualize a variable such as expression by distance to an anchor points.
+`@LLehner <https://github.com/LLehner>`__
```

Comment:
use bulletpoint to follow more closely the formatting in scanpy

Reply:
actually remove this altogether cause release notes already enough no need for changelog